clerk/instance_organization_permissions.go

```diff
@@ -24,3 +25,62 @@ func (s *InstanceService) ListOrganizationPermissions(params ListInstanceOrganiz
 	}
 	return response, nil
 }
+
+type CreateInstanceOrganizationPermissionParams struct {
```

Comment:
:upside_down_face: Can't we name this `CreateOrganizationPermissionsParams`?

Reply:
What if we introduce the same method on the organization resource/service as well? As those are under the same package, there will be a name conflict

Reply:
> What if we introduce the same method on the organization resource/service as well? As those are under the same package, there will be a name conflict

Yes let's keep it with `instance` in the name, at least for this iteration of the project.